### PR TITLE
Delete both EasyPlay and Mindurka due to conflicts

### PR DESCRIPTION
**Pull request by 4cr0-b47**
**[RUS]**
Из-за конфликтов, которые происходят между Ру и Миндуркой. Я думаю, что единственный способ решить данную проблему - удалить оба сервера из глобального списка.

**[ENG]**
Due to the conflicts that occur between Ru and Mindurka. I think that the only way to solve this problem is to remove both servers from the global list.

### CODE REVIEW

**Comment by 4cr0-b47** [comment]
Недавний пулл об снятии: https://github.com/Anuken/Mindustry/pull/5052
Пулл, где прямо говорилось от администрации миндурки про ру: https://github.com/Anuken/Mindustry/pull/5074

**Comment by osp54** [comment]
В данном случае, вы начинаете этот конфликт заново, ведь этот конфликт закончился, и миндурка и ру не конфликтуют между собой. Ваш ПР также проигнорит Анюк, он не любит срачи, конфликты, как было с прошлым пуллом об удалении миндурки. Прошу угомонится.

**Comment by Darkness6030** [comment]
Конфликт решен, мы сохраняем нейтралитет. Не вижу смысла в удалении серверов, ведь куда тогда пойдут игроки? 
На RCR? 
Но это один из самых плохо настроенных серверов в игре...

**Comment by Dioo0** [comment]
u kinda late

**Comment by Erisfiregamer1** [comment]
EasyPlay has barely conflicted with MIndurka, and plus they have rewritten their originally toxic rules.
MIndurka should do the same. Still, I think that only Mindurka should be removed.

**Review by Erisfiregamer1 — COMMENTED** [review]
put the easyplay back

**Comment by osp54** [comment]
> EasyPlay has barely conflicted with MIndurka, and plus they have rewritten their originally toxic rules.
> MIndurka should do the same. Still, I think that only Mindurka should be removed.

1You are so smart with us, but did you know that the conflict started EasyPlay.co, you really, when you do not know anything, eat to agrude, and you know, judging the server by conflicts, is silly.

**Comment by osp54** [comment]
> EasyPlay has barely conflicted with MIndurka, and plus they have rewritten their originally toxic rules.
> MIndurka should do the same. Still, I think that only Mindurka should be removed.

And yes you are toxic

**Comment by Erisfiregamer1** [comment]
toxic, eh
let me list some things I know:
1. EasyPlay tried to remove Mindurks due to bad things happening there. Reputable community moments (baseduser, and 1 or 2 others) said the pull was OK. You keep defending MinDurka, choosing now to smear EasyPlay's rep. Not helping.
2. This pull was an attempt to stop the drama. I agree well that something must be done, but the community is a bit in disagreement over it. We should get a consensus first.
3. When have I been toxic? Show me an example.

File: servers_v6.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   },
   {
     "name": "C.A.M.S.",
-    "address": ["routerchain.ddns.net", "nikochio.ddns.net", "easyplay.su"]
+    "address": ["routerchain.ddns.net", "nikochio.ddns.net"]
```
[the remark this answers is not in the document]
Comment (Erisfiregamer1) [reply servers_v6.json:24]: forget this.
if only MinDurka is removed, I feel worse will come.

**Comment by osp54** [comment]
> toxic, eh
> let me list some things I know:
> 
> 1. EasyPlay tried to remove Mindurks due to bad things happening there. Reputable community moments (baseduser, and 1 or 2 others) said the pull was OK. You keep defending MinDurka, choosing now to smear EasyPlay's rep. Not helping.
> 2. This pull was an attempt to stop the drama. I agree well that something must be done, but the community is a bit in disagreement over it. We should get a consensus first.
> 3. When have I been toxic? Show me an example.

It's sad to communicate with you, you scum which believes all fake information and spreads it.

**Comment by Erisfiregamer1** [comment]
I said show me an example (like from a pull), not just choosing to smear me worse.

**Comment by osp54** [comment]
> I said show me an example (like from a pull), not just choosing to smear me worse.
I shared my opinion about you, and I will not provide you with any evidence

**Comment by Erisfiregamer1** [comment]
without evidence you are just saying you don't like me very harshly. why not just say you do not like me and get over with?

**Comment by osp54** [comment]
I am ending my dialogue with you, because it is pointless to talk to you. Let the anuke decide what to do.

**Review by osp54 — CHANGES_REQUESTED** [review]
Bye Bye.

**Comment by summetdev** [comment]
Вам ещё не надоели эти детские войны между серверами?

**Comment by MHeMoTexHuK** [comment]
Calm, Anuke will ignore this pr
IMG_20210413_191542

**Comment by Erisfiregamer1** [comment]
20 messages. this pull is getting T H I C C

**Comment by Dioo0** [comment]
damn, just close this dumb pr

**Review by Volas171 — APPROVED** [review]

**Review by Volas171 — APPROVED** [review]

**Comment by Volas171** [comment]
wait what

File: servers_v6.json [hunk 2]
```diff
@@ -87,10 +87,6 @@
     "name": "Nydus",
     "address": ["v6.mindustry.nydus.app:6566"]
   },
-  {
-    "name": "MD Community",
-    "address": ["62.109.31.34", "185.13.47.140:1001"]
-  },
```
Comment (Volas171) [inline servers_v6.json:93]: leave mindurka, .ru just technically changed name to remove their own reputation ,

**Review by Volas171 — CHANGES_REQUESTED** [review]

**Comment by Volas171** [comment]
**ignore above really dont know what happened**

**Comment by bvcbvc** [comment]
this is dumb

**Pull request closed without merging**